src/inputs/Button/Button.stories.tsx

```diff
@@ -1,13 +1,40 @@
-// Button.stories.tsx
-import { Meta, Story } from "@storybook/preact";
-import { h } from "preact";
-import { Button } from "./Button";
-import * as React from "preact/compat";
-
+import { Meta, Story } from '@storybook/preact'
+import Button from './Button'
+import * as React from 'preact/compat'
+import { h } from 'preact'
 export default {
-  component: Button,
-} as Meta;
+	component: Button,
+	argTypes: {
+		variant: {
+		  options: ['primary', 'secondary'],
+		  control: { type: 'radio' },
+		},
+	  }
+} as Meta
+
+
+
+export const Primary: Story = args => (
+	<Button {...args} onClick={() => console.log('clicked')}>
+		Primary
+	</Button>
+)
+Primary.args = {
+	variant: 'primary',
+	disabled: false,
+	isLoading: false
+}
+
+export const Loading: Story = args => (
```

Comment:
**suggestion:** I would keep one Button story here (without splitting it for Loading, Disabled separately). It will be easier to maintain